cmd/utils.go

```diff
@@ -301,13 +341,16 @@ func getOperatorHome(config *RootCommandConfig) string {
 }
 
 func getProjectName(config *RootCommandConfig) (string, error) {
-	projectDir, err := getProjectDir(config)
+	dir, err := getProjectDir(config)
 	if err != nil {
 		return "my-project", err
 	}
-	projectName := strings.ToLower(filepath.Base(projectDir))
-	projectName = strings.ReplaceAll(projectName, "_", "-")
-	return projectName, nil
+
+	appsodyConfig := filepath.Join(dir, ConfigFile)
+	v := viper.New()
+	v.SetConfigFile(appsodyConfig)
+	err = v.ReadInConfig()
+	return v.GetString("project-name"), err
```

Comment:
@kylegc and I discussed a bit here. This is a get method - but... it may happen that someone manually manipulates the configuration file. So - we should do two things:
* check that the name is still compliant with the k8s regex. If it isn't, Error.log and error out.
* if we don't find anything (i.e. someone happily deleted the project name from the file) we should re-gen it (just like we did in `init`) and set it in the file. Warning.log("Your project name is now...").

Reply:
Done